frame/plasm-rewards/src/lib.rs

```diff
@@ -0,0 +1,428 @@
+//! # Plasm Staking Module
+//!
```

Comment:
What's about to insert here module header like https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L17

Reply:
Thinks of goods. 😄